tests/benchmarks/power_consumption/uart_polling/prj.conf

```diff
@@ -0,0 +1,17 @@
+# Enable runtime power management for peripheral
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
+
+# Enable logging
+# CONFIG_LOG=y
+
+CONFIG_PM=y
+CONFIG_PM_S2RAM=y
+CONFIG_POWEROFF=y
+CONFIG_PM_S2RAM_CUSTOM_MARKING=y
+# CONFIG_CONSOLE=n
+# CONFIG_UART_CONSOLE=n
+# CONFIG_SERIAL=n
+# CONFIG_GPIO=n
```

Comment:
Without gpio too?

Reply:
Removed all commented-out entries in prj.conf's